pkg/form/form.go

```diff
@@ -13,6 +13,10 @@ import (
 	"github.com/muesli/termenv"
 )
 
+const (
+	DefaultLimit = 100
```

Comment:
Can this be an `unexported` variable since it's only used in this package?

```suggestion
	defaultLimit = 100
```

Reply:
I couldn't find other default values, I did saw they are in the struct as defaults for the json, perhaps if this gets standarized this problem can be avoided next time, for example, right now the same thing would happen for port and host which can be comfortably defaulted to `localhost` and `5432`

Reply:
I've un-exported the constant as well